docs/getting-started/deploy.md

```diff
@@ -1,52 +1,17 @@
 ---
 id: deploy
-title: How to deploy your services
-sidebar_label: Deploy your services
+title: Deploy to Docker Desktop
```

Comment:
I don't think this doc should only be about "docker desktop" .Instead, it should still talk more generically about how to deploy services built with Amplication

Reply:
Deploying to Kubernetes is a whole different story. Client should have image registry and cluster, we do not describe this flow because we do not provide client with tools for it, for example, helm chart, CI workflow.